### DISCUSSION TITLE
Text comparison tool

### DISCUSSION THREAD

**Discussion by 170yt** [CorentinTh/it-tools#383] [Ideas]
I would love to see a new tool (similar to the newly added 'JSON diff' tool) that compares two texts and shows their differences. It could be really similar to the 'JSON diff' tool, but works with all kinds of texts (not just JSON). 

I must say, the work you've done so far in this project is truly impressive. Keep up the good work!

**Comment by syafi-dev** [comment]
Another vote for this. Hopefully this feature can be implemented.

**Comment by CorentinTh** [comment]
Done ✅ 
https://it-tools.tech/text-diff

![screely-1693061250172](https://github.com/CorentinTh/it-tools/assets/25065347/d95de8e3-5bcc-4183-94dc-36112eac246d)
